index.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module 'waveform-data' {
+
+    type WaveformDataChannelType = {
```

Comment:
The `Type` suffix is unnecessary, I'd prefer to name these `WaveformDataChannel` and `WaveformData`.